hackathon_site/dashboard/frontend/src/slices/order/adminOrderSlice.test.tsx

```diff
@@ -0,0 +1,97 @@
+import thunk, { ThunkDispatch } from "redux-thunk";
```

Comment:
Can you rename the file as adminOrderSlice.test.ts instead of with 'tsx'? Just for convention